core/src/test/java/tech/tablesaw/SortTest.java

```diff
@@ -149,4 +151,19 @@ public void createSortInvalidPrefixColumnDoesNotExist() {
 
     assertEquals("Unrecognized Column: '>col2'", thrown.getMessage());
   }
+
+  @Test
+  public void testGetOrderAndCreate0() throws Exception {
+    assertThrows(IllegalStateException.class, () -> {
+      Table table = Table.create("table1", IntColumn.create(" ales"));
+      AnalyticQuery query = AnalyticQuery.query().from(table)
+                                         .partitionBy("product", "region")
+                                         .orderBy("sales")
+                                         .rowsBetween()
+                                         .unboundedPreceding()
+                                         .andUnBoundedFollowing()
+                                         .sum("sales").as("sumSales")
+                                         .build();
```

Comment:
Codacy is complaining that `query` is unused
```suggestion
      AnalyticQuery.query()
          .from(table)
          .partitionBy("product", "region")
          .orderBy("sales")
          .rowsBetween()
          .unboundedPreceding()
          .andUnBoundedFollowing()
          .sum("sales").as("sumSales")
          .build();
```